test/unit/test_exports/test_txt_export.py

```diff
@@ -105,3 +123,33 @@ def test_there_is_a_next_time(self, my_export):
     def test_last(self, my_export):
         assert my_export.when_is_next_time(3) is None
         assert my_export.when_is_next_time(4) is None
+
+
+class TestIsItFirstTimeToExport:
```

Comment:
Could do with some doc strings here

Reply:
Kinda hard to document a test class, you can document the test functions however.